### DISCUSSION TITLE
v0.81.0 buggy version

### DISCUSSION THREAD

**Discussion by lin-ycv** [lin-ycv/EverythingPowerToys#107] [Announcements]
After the initial release of v0.81.0, I noticed I uploaded the wrong files, so I changed them out, but before I noticed and did the swap, a dozen or so people already install the buggy version.

If you're one of the dozen so people that downloaded the buggy version, just download v0.81.0 again.

**Comment by kai-earthrise** [comment]
Can you confirm the fixed version is live? I just installed for the first time and am getting an exception on each use along the lines of

> System.EntryPointNotFoundException: Unable to find an entry point named 'Everything_IncRunCountFromFileName' in DLL 'Everything64.dll'.
   at Community.PowerToys.Run.Plugin.Everything.Interop.NativeMethods.<Everything_IncRunCountFromFileName>g____PInvoke|7_0(UInt16* __lpFileName_native)

I don't know if I'm encountering a real bug or if I missed something in the install and setup.

**Reply by lin-ycv** [reply]
This is related to #108, are you on arm or x64?

**Reply by kai-earthrise** [reply]
I'm on x64. I should brush up on my Chinese.

**Reply by lin-ycv** [reply]
It seems to be an issue with me changing to LibraryImport from dllimport
Will do some tests and push a fix

**Reply by lin-ycv** [reply]
Just to confirm, this does not happen on v0.80.0 of this plugin?

**Reply by kai-earthrise** [reply]
Correct, I just installed 0.80.0 and it works flawlessly, no errors.
